src/main/java/lol/fairplay/ghidraapple/analysis/objcclasses/OCTypeInjectorAnalyzer.kt

```diff
@@ -90,14 +90,30 @@ class OCTypeInjectorAnalyzer : AbstractAnalyzer(NAME, DESCRIPTION, AnalyzerType.
 
             taskMonitor.checkCancelled()
 
-            val allocCalls: List<PcodeOpAST> = decompiled.highFunction.pcodeOps.asSequence()
+            val highFunction = decompiled.highFunction
+            if (highFunction == null) {
+                Msg.error(this, "Failed to inject types: highFunction is null for ${function.name}")
```

Comment:
This kind of failure seems pretty bad and should be reported via the analysis log instead.
If anything has been logged to it, a popup with the messages will show after the analysis is finished